spec/spec_helper.rb

```diff
@@ -25,6 +25,7 @@
 
 SimpleCov.start do
   add_filter '/spec/'
+  add_filter '/lib/faraday/adapter/em_http_ssl_patch'
```

Comment:
I really don't think this is a good idea.
If you've decided you don't want to test it, fine, but don't hide the fact that it's not tested.

Reply:
After doing some research into what this is, I feel strongly that it should be tested. 
So I'll add it to my list of things to do.

Reply:
I've already agreed with the other maintainers that the SSL patch will be deprecated in v1.0 and dropped in future versions, hence my decision not to spend any time test-covering it, also because it would be hard to do it with unit-tests.

The current plan is to eventually cover it in some sort of integration tests, which we can add later. If you want to help in that direction then you're more than welcome 😃.

That said, I saw your other PR so it seems like there are other files that were ignored, so if we agree on merging that one, it makes sense to keep the file in